### ISSUE TITLE
cylc-review: line numbers misaglined for output longer than page width

### ISSUE THREAD

**Issue by arjclark** [cylc/cylc-flow#3185]
**Brief description**

In cylc review (checked at cylc 7.8.1) display of job logs where a line has gone longer than the width of the box in which it is rendering the output it results in later misaligned line numbers. This seems to be a result of the longlines being wrapped in the main box but the line numbers entry not being spaced accordingly (see screenshot below).

**Steps to reproduce the bug**
Can be reproduced by printing a longline (as sometime seen when printing an output path).

**Expected behavior**
Either, there should be a scroll bar on the window so you just scroll along the output (i.e. no line wrapping) or the line number entries on the left hand side should visibly align with where the text is (N.B. clicking on the numbers does highlight the Nth line in full correctly so this is a presentation thing).

**Screenshots**
Screenshot of problem:

![image](https://user-images.githubusercontent.com/2562650/59017468-7cd08880-883b-11e9-8199-6a3b2d592d3b.png)

Screenshot demonstrating that the lines are at least correctly associated with the numbers (highlights the correct line in full):

![image](https://user-images.githubusercontent.com/2562650/59017823-6b3bb080-883c-11e9-85b0-29caf06d6623.png)

Screenshot to illustrate just how small the font needs to be in the specific case seen in order for the lines to line up (though changing your font size should not be a solution as it ignores accessibility issues):

![image](https://user-images.githubusercontent.com/2562650/59017896-90c8ba00-883c-11e9-8194-7eb98a480456.png)

**Comment by matthewrmshin** [comment]
Added to next-release, but is likely to be pushed back to a follow on milestone in the 7.8.x series. Note: Cylc Review has been removed from cylc-flow master, and will be reimplemented in the Cylc 8 time frame. This issue may or may not apply to the presentation framework used in the new UI.

**Comment by wxtim** [comment]
@arjclark  - I've just been investigating this, but I don't seem to be able to reproduce it on 7.8.x branch with workflow task 
```
        script = """
            python -c "foo = '-foo' * 8000; print(foo); bar = '@bar-' * 77;print(bar)"
            for i in {1..99}; do
                printf "bazzz"
            done
        """
```

**Comment by arjclark** [comment]
@wxtim - as this is nearly the second anniversary of raising the problem the logs in question demonstrating the problem no longer exist and having moved jobs I no longer run suites as a daily activity. I leave it up to yourselves to decide what to to about this - as @matthewrmshin alludes maybe whatever causes this may be a non-issue with cylc8 maybe fixing cylc-review is moot at this point.

**Comment by wxtim** [comment]
> @wxtim - as this is nearly the second anniversary of raising the problem the logs in question demonstrating the problem no longer exist and having moved jobs I no longer run suites as a daily activity. I leave it up to yourselves to decide what to to about this - as @matthewrmshin alludes maybe whatever causes this may be a non-issue with cylc8 maybe fixing cylc-review is moot at this point.

If I could have replicated I'd have had a go at fixing. Hopefully C8 will make this a moot point in the near future.

**Comment by dpmatthews** [comment]
I assume this was fixed by #3437
